src/r_util.cpp

```diff
@@ -521,6 +554,7 @@ namespace rhost {
             { "Microsoft.R.Host::Call.destroy_blob", (DL_FUNC)destroy_blob, 1 },
             { "Microsoft.R.Host::Call.get_file_lock_state", (DL_FUNC)get_file_lock_state, 1 },
             { "Microsoft.R.Host::Call.fetch_file", (DL_FUNC)fetch_file, 1 },
+            { "Microsoft.R.Host::Call.save_project", (DL_FUNC)save_project, 4 },
```

Comment:
I'm not sure how much we can provide even. Rf_error just takes an error message string, and exceptions_to_errors will automatically translate the exception message string to that.

---
In reply to: [74849277](https://github.com/Microsoft/R-Host/pull/78#discussion_r74849277) [](ancestors = 74849277,74654678)